embedchain/embedchain.py

```diff
@@ -114,19 +150,28 @@ def load_and_embed(self, loader: BaseLoader, chunker: BaseChunker, src, metadata
             ids = list(data_dict.keys())
             documents, metadatas = zip(*data_dict.values())
 
-        # Add app id in metadatas so that they can be queried on later
-        if self.config.id is not None:
-            metadatas = [{**m, "app_id": self.config.id} for m in metadatas]
+        # Loop though all metadatas and add extras.
+        new_metadatas = []
```

Comment:
we should append the `Successfully saved https://nav.al/agi. New chunks count` print statement. We should show the data type deduced here, so that everyone has visibility of the data type and can corrections if needed.

I know about the logging part, but a lot of users might not use logging. Hence the ask

Reply:
Good idea

about the logging: warning and above are displayed by default. lower has to be enabled.